### ISSUE TITLE
`qiita webserver start` error message improvements

### ISSUE THREAD

**Issue by jairideout** [biocore/qiita#915]
From a fresh Qiita install, I get the following error:

```
$ qiita webserver start
Traceback (most recent call last):
  File "/Users/jairideout/.virtualenvs/qiita/bin/qiita", line 10, in <module>
    execfile(__file__)
  File "/Users/jairideout/dev/qiita/scripts/qiita", line 20, in <module>
    from qiita_db.commands import (load_sample_template_from_cmd,
  File "/Users/jairideout/dev/qiita/qiita_db/commands.py", line 23, in <module>
    from .metadata_template import (SampleTemplate, PrepTemplate,
  File "/Users/jairideout/dev/qiita/qiita_db/metadata_template.py", line 1138, in <module>
    class SampleTemplate(MetadataTemplate):
  File "/Users/jairideout/dev/qiita/qiita_db/metadata_template.py", line 1154, in SampleTemplate
    'required_sample_info_status_id': get_required_sample_info_status()}
  File "/Users/jairideout/dev/qiita/qiita_db/util.py", line 217, in get_required_sample_info_status
    con = SQLConnectionHandler()
  File "/Users/jairideout/dev/qiita/qiita_db/sql_connection.py", line 120, in __init__
    self._open_connection()
  File "/Users/jairideout/dev/qiita/qiita_db/sql_connection.py", line 155, in _open_connection
    raise RuntimeError("Cannot connect to database: %s" % str(e))
RuntimeError: Cannot connect to database: FATAL:  role "postgres" does not exist
```

It would be helpful to point users to their Qiita config file and specify what setting(s) need to be added/modified.

**Comment by wdwvt1** [comment]
addressed in #957

**Comment by wdwvt1** [comment]
solved in #957.
